src/SDK/SmartDetectors.ProjectTemplate/LogSearchAlert.cs

```diff
@@ -23,20 +25,20 @@ public class $alertName$ : Alert
         /// <summary>
         /// Gets or sets the resource name
         /// </summary>
-        [AlertPredicateProperty]
-        [AlertPresentationProperty(AlertPresentationSection.Property, "Resource name", Order = 1, InfoBalloon = "The resource name")]
+        [PredicateProperty]
+        [TextProperty("Resource name", Order = 1)]
         public string ResourceName { get; set; }
 
         /// <summary>
         /// Gets or sets the number of records
         /// </summary>
-        [AlertPresentationProperty(AlertPresentationSection.Property, "Number of records", Order = 2, InfoBalloon = "The number of records")]
+        [TextProperty("Number of records", Order = 2)]
         public int Count { get; set; }
 
         /// <summary>
         /// Gets the query that displays the number of records over time
         /// </summary>
-        [AlertPresentationProperty(AlertPresentationSection.Chart, "Number of records over time for {ResourceName}", Order = 3, InfoBalloon = "The number of records over time")]
-        public string CountChart => $@"$query$";
+        [ChartProperty("Number of records over time for {ResourceName}", ChartType.LineChart, ChartAxisType.DateAxis, ChartAxisType.NumberAxis, Order = 3)]
+        public List<ChartPoint> CountChart { get; set; }
```

Comment:
maybe be a real gever and add a table property as well

Reply:
not for this change - it's the bare minimum to make it work

---
In reply to: [280451874](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/62#discussion_r280451874) [](ancestors = 280451874)